src/sheet-common.tsx

```diff
@@ -237,6 +237,7 @@ const Sheet = React.forwardRef<HTMLDivElement, SheetProps>(({onClose, children,
                     {...dragableSheetProps}
                     {...getPrefixedDataAttributes(dataAttributes, 'Sheet')}
                     ref={ref}
+                    tabIndex={-1}
```

Comment:
I think the focus should be moved to the element with role=dialog

Reply:
You're right. Done :)